### ISSUE TITLE
Support for adding tracks after the currently playing

### ISSUE THREAD

**Issue by carnager** [Spotifyd/spotifyd#23]
Sometimes one wants to listen to some track before continuing with the actual playlist. 
some kind of "insert" command would be handy for this. maybe even with the ability
to stack several songs after another, before regular playlist continues.

is this possible using libspotify?

**Comment by SimonPersson** [comment]
It is certainly possible. Depending on #20, it should be possible to achieve this with two commands no? Something like `sc qadd n && sc move $LAST_TRACK $CUR_TRACK_PLUS_ONE`, no? That way we don't need to add so many commands to spotifyd. Last track and current track should be easy to get with a script.

**Comment by carnager** [comment]
sure, if you provide such variables that would work just fine

**Comment by mastrone** [comment]
In my opinion it would be better to implement a double queue: playlist and normal queue.
When you are playing a playlist it follows it only when the queue is empty. Following this approach you can let a playlist play and insert new songs adding them to the queue. When the queue is empty it will resume playing the initial playlist.

Hopefully I will make sense :)

**Comment by SimonPersson** [comment]
As Spotifyd had to be rewritten entirely from scratch, this bug report is no longer relevant. Feel free to try out the new version and report any issues.

Closing.
